INSTALL.md

```diff
@@ -7,7 +7,9 @@ tested on FreeBSD, GNU/Linux and OS X.
 ## Build dependencies
 
 1. **C++ compiler**, to compile the sources. (Shall support
-   [C++17 dialect](https://en.cppreference.com/w/cpp/compiler_support#cpp17).)
```

Comment:
Feels like this level of details is needed to explain in a separate section maybe?